js/repl.ts

```diff
@@ -6,7 +6,9 @@ import * as deno from "./deno";
 import { close } from "./files";
 import * as dispatch from "./dispatch";
 import { exit } from "./os";
-import { window } from "./globals";
+import { globalEval } from "./global_eval";
+
+const window = globalEval("this");
```

Comment:
doing that won't cause window to miss all properties assigned to it in `globals.ts`? I think we would like to use most of them in REPL

Reply:
Valid question ... However, I'm going to land this anyway because I need the fix and because this is passing the tests. If this indeed breaks `window` properties, we should add tests that nail that behavior down.

Reply:
It doesn't just rebased it against my REPL branch and everything works as expected.